public/app/features/dashboard/components/GenAI/GenAIHistory.tsx

```diff
@@ -0,0 +1,203 @@
+    }
+  }, [streamStatus, reply, updateHistory]);
+
+  useEffect(() => {
+    if (error) {
+      setShowError(true);
+    }
+
+    if (streamStatus === StreamStatus.GENERATING) {
+      setShowError(false);
+    }
+  }, [error, streamStatus]);
+
+  const onSubmitCustomFeedback = (text: string) => {
+    onGenerateWithFeedback(text);
+    reportInteraction(AutoGenerateItem.customFeedback, { customFeedback: text });
+  };
+
+  const onApply = () => {
+    onApplySuggestion(history[currentIndex - 1]);
+  };
+
+  const onNavigate = (index: number) => {
+    setCurrentIndex(index);
+    reportInteraction(index > currentIndex ? AutoGenerateItem.backHistoryItem : AutoGenerateItem.forwardHistoryItem);
+  };
+
+  const onGenerateWithFeedback = (suggestion: string | QuickFeedbackType) => {
+    if (suggestion !== QuickFeedbackType.Regenerate) {
+      messages = [...messages, ...getFeedbackMessage(history[currentIndex], suggestion)];
+    }
+
+    setMessages(messages);
+
+    if (suggestion in QuickFeedbackType) {
+      reportInteraction(AutoGenerateItem.quickFeedback, { quickFeedbackItem: suggestion });
+    }
+  };
+
+  return (
+    <div className={styles.container}>
+      {showError && (
+        <div>
+          <Alert title="">
+            <VerticalGroup>
+              <div>Sorry, I was unable to complete your request. Please try again.</div>
+            </VerticalGroup>
+          </Alert>
+        </div>
+      )}
+
+      <Input
+        placeholder="Tell AI what to do next..."
+        suffix={
+          <IconButton
+            name="corner-down-right-alt"
+            variant="secondary"
+            aria-label="Send custom suggestion"
+            onClick={() => onSubmitCustomFeedback(customFeedback)}
+            disabled={customFeedback === ''}
+          />
+        }
+        value={customFeedback}
+        onChange={(e) => setCustomPrompt(e.currentTarget.value)}
+        onKeyDown={(e) => e.key === 'Enter' && onSubmitCustomFeedback(customFeedback)}
+      />
+      <div className={styles.actions}>
+        <QuickFeedback onSuggestionClick={onGenerateWithFeedback} isGenerating={isStreamGenerating} />
+        <GenerationHistoryCarousel
+          history={history}
+          index={currentIndex}
+          onNavigate={onNavigate}
+          reply={reply.replace(/^"|"$/g, '')}
+          streamStatus={streamStatus}
+        />
+      </div>
+      <div className={styles.applySuggestion}>
+        <HorizontalGroup justify={'flex-end'}>
+          {isStreamGenerating && <Spinner />}
+          <Button onClick={onApply} disabled={isStreamGenerating}>
+            Apply
+          </Button>
+        </HorizontalGroup>
+      </div>
+      <div className={styles.footer}>
+        <Icon name="exclamation-circle" aria-label="exclamation-circle" className={styles.infoColor} />
+        <Text variant="bodySmall" element="p" color="secondary">
+          Be aware that this content was AI-generated.{' '}
```

Comment:
Current wording is a little awkward IMO - just telling them that content is ai generated with an additional link is good enough IMO

```suggestion
          This content is AI-generated.{' '}
```